scripts/src/tasks/clean.js

```diff
@@ -1,9 +1,31 @@
 // @ts-check
 
+import { Command } from 'clipanion';
+import * as fs from 'fs';
```

Comment:
```suggestion
import * as fs from 'node:fs';
```

Reply:
This syntax (node:fs) doesn't seem to work in the repo right now, I need to look into it.